### ISSUE TITLE
looping in cheat mode does not work

### ISSUE THREAD

**Issue by MrZorG33** [We-the-People-civ4col-mod/Mod#682]
there is an idea to figure out why the looping of players in the "[Ctrl + Alt + Z](https://civ-wiki.de/wiki/Tipps_zum_Modden_(TAC))" cheat mode does not work. after switching between all players, you should return to the player with which you started your game. but now you are stuck on the "Wild Animals" player - switching does not work further. this mode is sometimes useful for testing new releases. so if anyone can solve this problem that would be great.

**Comment by raystuttgart** [comment]
**By the way:**
"SHIFT + ALT + Z" also allows to go back.

So maybe the "cycling" is not even needed.

**Comment by raystuttgart** [comment]
@MrZorG33 
Please also try if you can cycle in Vanilla.
(It would be helpful to know if it works there.)

**Comment by MrZorG33** [comment]
yes, it works. however, I did not find such a keyboard shortcut in the article. probably this is again the "secrets" of the old modders)

however, it took me quite a while. if someone would like to create a list for this mode, where they could choose the player to switch to, that would be great. so let's leave it open for now.

**Comment by raystuttgart** [comment]
There is a way to view all Cheats available:

1) „Shift + Ö“ (For English Keyboard: I think it is the "chipotle": ^)
2) Type: help ---> Press: Enter

It should then list all available modder cheats.

**Comment by raystuttgart** [comment]
Those are the console cheats:
![Civ4ScreenShot0001](https://user-images.githubusercontent.com/59391125/212541277-a790b122-ba76-450a-aaa1-cd5347290b8c.JPG)

But it is actually pretty pointless today to use the console, because the Cheat Menus that were built in TAC and AOD2 are much more convenient and can even do more.
